### DISCUSSION TITLE
Cascader component?

### DISCUSSION THREAD

**Discussion by mazipan** [mantinedev/mantine#1325] [Feature requests]
<img width="406" alt="Screen Shot 2022-04-25 at 21 47 21" src="https://user-images.githubusercontent.com/7221389/165113877-1bc6f65a-2a53-4de8-a428-ee8fedd1844e.png">

Ant Design has a Cascader component: https://ant.design/components/cascader/
The way it shows multiple context menu maybe can be replicated by Radix-UI context menu: https://www.radix-ui.com/docs/primitives/components/context-menu

**Comment by rtivital** [comment]
We are not planning to add such component any time soon
